compiler/src/iree/compiler/GlobalOptimization/PropagateLinalgTranspose.cpp

```diff
@@ -137,18 +137,22 @@ static void specializeGenericTransposeOp(RewriterBase &rewriter,
 /// generalizing is allowed. Otherwise if the `op` is a linalg::GenericOp,
 /// then just return the generic op.
 static FailureOr<linalg::GenericOp>
-getGenericOpOrGeneralizeContraction(RewriterBase &rewriter, Operation *op,
-                                    bool allowGeneralizing) {
+getAllowedGenericOpOrGenerializeNamedOp(RewriterBase &rewriter, Operation *op,
```

Comment:
typo: Generialize -> Generalize

Please also update the above comment. I think it could be something like below, and we can move the TODO to here.

```
// Returns the `op` if it is a linalg::GenericOp. If it is a named op and `allowGeneralizing` is true, returns the generalized op. Otherwise, returns failure.
// TODO: Right now convolutions are not allowed due to fragility around handling of convolutions.
```

Reply:
Nice catch! Thanks for providing the well written comments.